packages.json

```diff
@@ -4,20 +4,16 @@
             "all": [
                 "alsa-firmware",
                 "android-udev-rules",
-                "apr",
-                "apr-util",
```

Comment:
APR was added specifically to support installations of Davinci Resolve ( #91 ) and was a nice example of a user providing a clean request for package support or alternative documentations. 

There's no harm to keeping this, but removing it is reversing a past decision to save... 246k on a download? 

I'm not sure I would support adding this today, but I'm not really in favor of removing support without a strong reason.

Reply:
> save... 246k on a download?

Image size is not the purpose of this PR :slightly_smiling_face: